.pre-commit-config.yaml

```diff
@@ -1,5 +1,11 @@
 repos:
 - repo: https://github.com/psf/black
-  rev: 19.3b0
+  rev: 19.10b0
   hooks:
   - id: black
+
+- repo: https://github.com/asottile/blacken-docs
+  rev: v1.3.0
```

Comment:
Again, I'm disinclined to introduce more toil by adding a version that needs to be repeatedly updated.

Reply:
Running `pre-commit autoupdate` from time to time will update them all.

Because they're pinned, it means when there's a new release of something, it won't cause sudden surprises.